charts/coralogix-operator/templates/alert_crs_not_synced.yaml

```
@@ -0,0 +1,30 @@
+{{- if .Values.alerts.enabled }}
+apiVersion: coralogix.com/v1beta1
+kind: Alert
+metadata:
+  labels:
+  {{ include "coralogixOperator.labels" . | nindent 4 }}
+  name: {{ include "coralogixOperator.fullname" . }}-crs-not-synced
+spec:
+  name: {{ include "coralogixOperator.fullname" . }}-crs-not-synced
+  description: Fires when the ratio of RemoteUnsynced state is more than 5%, grouped by kind. This indicates a partial sync failure between Kubernetes and Coralogix for a specific resource type or namespace.
```

```suggestion
  description: Fires when more than 5% of the custom resources are not synced, grouped by kind.
```
The rest is too much for the description.